Assets/Scripts/Gameplay/GameState/ServerBossRoomState.cs

```diff
@@ -95,37 +89,24 @@ private void ServerLoaded()
             }
         }
 
-        private void ClientLoaded(ulong clientId, string sceneName)
```

Comment:
This was recently removed here: https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/pull/691/files.

It's a less exhaustive check (no scene index check), but this approach works since by composition this class only exists in BossRoom scene.